packages/debs/utils/helper_function.sh

```diff
@@ -28,8 +28,10 @@ setup_build(){
     sed -i "s:RELEASE:${REVISION}:g" ${sources_dir}/debian/changelog
     sed -i "s:export JOBS=.*:export JOBS=${JOBS}:g" ${sources_dir}/debian/rules
     sed -i "s:export DEBUG_ENABLED=.*:export DEBUG_ENABLED=${debug}:g" ${sources_dir}/debian/rules
-    sed -i "s#export PATH=.*#export PATH=/usr/local/gcc-5.5.0/bin:${PATH}#g" ${sources_dir}/debian/rules
-    sed -i "s#export LD_LIBRARY_PATH=.*#export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}#g" ${sources_dir}/debian/rules
+    sed -i "s#export PATH=.*#export PATH=/usr/local/gcc-14.3.0/bin:${PATH}#g" ${sources_dir}/debian/rules
+    sed -i "s#export LD_LIBRARY_PATH=.*#export LD_LIBRARY_PATH=/usr/local/gcc-14.3.0/lib64#g" ${sources_dir}/debian/rules
```

Comment:
Could we keep the previous variable instead of hardcoding it again?
```suggestion
    sed -i "s#export LD_LIBRARY_PATH=.*#export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}#g" ${sources_dir}/debian/rules
```

Reply:
Addressed in: https://github.com/wazuh/wazuh/pull/31813/commits/cce23ec1a2c31efe3d4ab517a27b18d90b07e605